src/opnsense/mvc/app/models/OPNsense/Proxy/Proxy.xml

```diff
@@ -69,6 +69,17 @@
                 <default>0</default>
                 <Required>N</Required>
             </suppressVersion>
+            <VisibleEmail type="EmailField">
+                <default>admin@localhost</default>
+                <Required>N</Required>
+                <ValidationMessage>Please enter a valid email address.</ValidationMessage>
+            </VisibleEmail>
+            <VisibleHostname type="TextField">
+                <default>localhost</default>
+                <Required>N</Required>
+                <mask>/^([0-9a-zA-Z.,_\-]){0,1024}$/u</mask>
```

Comment:
I think that "-" only needs to be written as "\-" when it would imply a sequence, but you don't want that ("[0-9]" vs. "[0\-9]" or rather "[09-]"). It's just cosmetic. :)